packages/cli/src/commands/build.ts

```diff
@@ -133,6 +143,7 @@ export class BuildCommand implements Command {
       ProjectBuildOptions {
     const buildOptions: ProjectBuildOptions = {};
     const validBuildOptions = new Set(this.args.map(({name}) => name));
+    validBuildOptions.delete(autoBasePath);  // This flag is special.
```

Comment:
`// This flag is special. It corresponds to the top-level config, not an individual build.`